worker/firewaller/firewaller.go

```diff
@@ -109,14 +53,18 @@ type Config struct {
 
 	CredentialAPI common.CredentialAPI
 
-	// TODO: (jack-w-shaw) Drop these once we move tests to mocks based
+	// These are used to coordinate gomock tests.
+
```

Comment:
Can these not de dropped? Would a DoAndReturn on the mocks not work here instead? 😭

Reply:
no because flushModel() as one example calls multiple methods like OpenModelPorts, CloseModelPorts only if needed so there's no reliable mocked method to hang the DoAndReturn off